modules/developers-guide/pages/concepts/data-centers.adoc

```diff
@@ -30,14 +30,14 @@ In this simplified scenario:
 * Each data center has hardware supplied by multiple node providers.
 * Any single node provider might have equipment in multiple data centers.
 
-Although this example represents one subnet with nodes in multiple data centers, any of the nodes could be moved out of this subnet to form a new subnet, if needed. 
-Changes to the network topology are managed through the {IC} governance system called the **Network Nervous System** (NNS). 
+Although this example represents one blockchain subnet with nodes in multiple data centers, any of the nodes could be moved out of this blockchain subnet to form a new blockchain subnet, if needed.
```

Comment:
subnet blockchain  and not blockchain subnet

Reply:
done